example/server_transform/server_transform.c

```diff
@@ -19,37 +39,7 @@
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
- */
-
-/* server-transform.c:  an example program that sends response content
- *                      to a server to be transformed, and sends the
```

Comment:
Yes, I thought it better to put it where Doxygen would see it, since it's the comment is already Doxygen formatted.